README.rst

```diff
@@ -5,9 +5,108 @@ Python language bindings for ev3dev
     :target: https://travis-ci.org/rhempel/ev3dev-lang-python
 .. image:: https://readthedocs.org/projects/python-ev3dev/badge/?version=latest
     :target: http://python-ev3dev.readthedocs.org/en/latest/?badge=latest
-    :alt: Documentation Statu
+    :alt: Documentation Status
 
-This is a python library implementing unified interface for ev3dev_ devices.
+A Python library implementing unified interface for ev3dev_ devices.
 
```

Comment:
How about adding a couple of examples here? Something like

Usage
--------

Run large motor connected to output port A for 3 seconds:
```py
>>> import ev3dev.ev3 as ev3
>>> m = ev3.LargeMotor('outA')
>>> m.connected
True
>>> m.run_timed(time_sp=3000, duty_cycle_sp=75)
```

Make your robot speak:
```py
>>> ev3.Sound.speak('Welcome to the EV3 project!').wait()
```

Use left led to indicate touch sensor state:
```py
>>> ts = ev3.TouchSensor()
>>> while True:
...    ev3.Leds.set_color(ev3.Leds.LEFT, ts.value() and ev3.Leds.RED or ev3.Leds.GREEN) 
```